conf/carbon.properties.template

```diff
@@ -52,7 +52,7 @@ carbon.compaction.level.threshold=4,3
 #default size (in MB) for major compaction to be triggered
 carbon.major.compaction.size=1024
 ######## Query Configuration ########
-#Number of cores to be used while querying
+#Number of cores to be used for loading blocks when execute query
```

Comment:
It is not just for query, compaction also need this parameter. 
I think a better description is "Number of cores to be used for loading index into memory"